docker/runTests.sh

```diff
@@ -2,4 +2,4 @@
 
 nohup mvn clean compile exec:java@catalogue-server & 
 sleep 40
-mvn clean test
\ No newline at end of file
+mvn clean test checkstyle:check pmd:check
```

Comment:
```suggestion
mvn clean test checkstyle:check pmd:pmd
```
use pmd instead of check